main/src/library/String.flix

```diff
@@ -71,4 +71,10 @@ namespace String {
     ///
     pub def trim(s: Str): Str = native method java.lang.String.trim(s) as Str
 
+    ///
+    /// Get the system line separator.
+    ///
+    pub def lineSeparator() : Str =
```

Comment:
I wonder if it would make sense to put this into some other namespaces? I guess we can leave it here for now...